types/minecraft-pinger/index.d.ts

```diff
@@ -0,0 +1,135 @@
+// Type definitions for minecraft-pinger 1.0
+// Project: https://github.com/dennisbruner/node-minecraft-pinger
+// Definitions by: Grayson-code <https://github.com/Grayson-code>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface pingPromiseInterface {
```

Comment:
this is called either `data` or `result` directly in the source code:
this is called `result` in the readme (public api):
https://github.com/dennisbruner/node-minecraft-pinger#example
Please use CamelCase notation for interfaces, most common notation pattern in JS:
https://developer.mozilla.org/en-US/docs/Web/API#interfaces